From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Cheng Zhao <zcbenz@gmail.com>
Date: Thu, 4 Oct 2018 14:57:02 -0700
Subject: fix: add back virtual methods in AccessiblePaneView

Mark SetPaneFocus and RemovePaneFocus as virtual in AccessiblePaneView, as we
need to override them in MenuBar.

Pending upstream patch: https://crrev.com/c/1959189

diff --git a/ui/views/accessible_pane_view.h b/ui/views/accessible_pane_view.h
index 4a4be84bf8d8b626ddc367506f61e9fb8cbefe60..81e6d6ac55eeae44337f336dc61bc1b021b60f25 100644
--- a/ui/views/accessible_pane_view.h
+++ b/ui/views/accessible_pane_view.h
@@ -36,7 +36,7 @@ class VIEWS_EXPORT AccessiblePaneView : public View,
   // If |initial_focus| is not NULL, that control will get
   // the initial focus, if it's enabled and focusable. Returns true if
   // the pane was able to receive focus.
-  bool SetPaneFocus(View* initial_focus);
+  virtual bool SetPaneFocus(View* initial_focus);
 
   bool pane_has_focus() const { return pane_has_focus_; }
 
@@ -84,7 +84,7 @@ class VIEWS_EXPORT AccessiblePaneView : public View,
   bool ContainsForFocusSearch(View* root, const View* v);
 
   // Remove pane focus.
-  void RemovePaneFocus();
+  virtual void RemovePaneFocus();
 
   View* GetFirstFocusableChild();
   View* GetLastFocusableChild();
